.konflux/bundle/overlay/pin_images.in.yaml

```diff
@@ -0,0 +1,5 @@
+# Do not modify the manager 'key' value: 'manager'
+- key: manager
+  source: quay.io/openshift-kni/numaresources-operator:4.20.999-snapshot
```

Comment:
the input source here is unusual but I assume this is just the placeholder this repo is using?

Reply:
yes, this is what we used to replace with render templates so keeping this for consistency .